### PR TITLE
Fix documentation typo & do not namespace keyframes 

### LINKED ISSUE #1: Support for styled-components keyframes
Originally reported by @genedwards,  moved from other repo.

Issue: the styled-components module exports a `keyframe` function. https://www.styled-components.com/docs/basics#animations

Example usage:
```Javascript
const rotate360 = keyframes`
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
`;

const Rotate = styled.div`
  animation: ${rotate360} 2s linear infinite;
`;
```

The generated CSS from the above example looks like the following:
```CSS
@-webkit-keyframeskrrMtA{
  .<cssNamespace> from{
    -webkit-transform:rotate(0deg);
    -ms-transform:rotate(0deg);
    transform:rotate(0deg);
  }
  .<cssNamespace> to{
    -webkit-transform:rotate(360deg);
    -ms-transform:rotate(360deg);
    transform:rotate(360deg);
  }
}
@keyframes krrMtA{
  .<cssNamespace> from{
    -webkit-transform:rotate(0deg);
    -ms-transform:rotate(0deg);
    transform:rotate(0deg);
  }
  .<cssNamespace> to{
    -webkit-transform:rotate(360deg);
    -ms-transform:rotate(360deg);
    transform:rotate(360deg);
  }
}
```

### PR DESCRIPTION
**Pull request by prevostc**
I wanted to update this plugin with a keyframe prefix, but looking at styled-components `styled.keyframes`, I realized it makes no sense :D 

I fix the README on my way to this conclusion.

Closes #1

### CODE REVIEW

**Comment by prevostc** [comment]
I did fixed something related to keyframes in the last commit, i'll update this PR title

**Comment by nvanselow** [comment]
Great! I just saw the bug fix come through. I'll have to take a closer look at that part sometime later today.

File: src/visitors/cssNamespace.js [hunk 1]
```diff
@@ -16,11 +16,11 @@ const getCssNamespace = state => {
 
 export default (path, state) => {
   const cssNamespace = getCssNamespace(state);
-
   if (
     isStyled(path.node.tag, state) &&
     path.node.quasi.quasis[0].value.cooked &&
-    !path.node.quasi.quasis[0].value.cooked.startsWith(`\n${cssNamespace} {`)
+    !path.node.quasi.quasis[0].value.cooked.startsWith(`\n${cssNamespace} {`) &&
+    path.node.tag.property.name !== 'keyframes'
```
Comment (genedwards) [inline src/visitors/cssNamespace.js]: Please add a sanity check here for `path.node.tag.property && ...`

Comment (prevostc) [reply src/visitors/cssNamespace.js]: I'm sorry I don't know babel internals very well, I don't know what to check :s 
`path.node.tag` is used  just above on line 20, should I do `... && (path.node.tag.property ?
 path.node.tag.property.name !== 'keyframes' : true)`

**Comment by prevostc** [comment]
Good call @genedwards I added a test case and fixed the issue :+1:

File: src/tests/cssNamespace.test.js [hunk 2]
```diff
@@ -44,6 +44,17 @@ pluginTester({
         __dirname,
         './fixtures/nested_helper_styled_component.js'
       )
+    },
+    {
+      title: 'Do not namespace keyframes',
```
Comment (genedwards) [inline src/tests/cssNamespace.test.js]: Can we keep this consistent with the others?
`'does not add namespace to keyframes'`

File: src/tests/cssNamespace.test.js [hunk 3]
```diff
@@ -44,6 +44,17 @@ pluginTester({
...
+      pluginOptions: {
+        cssNamespace: 'specific'
+      },
+      fixture: path.join(__dirname, './fixtures/keyframes_styled_component.js')
+    },
+    {
+      title: 'Namespace for extended components',
```
Comment (genedwards) [inline src/tests/cssNamespace.test.js]: same here: `'adds namespace to extended styled-components'`

**Comment by prevostc** [comment]
I Agree, consistency is important @genedwards

**Comment by prevostc** [comment]
Thanks a lot for all the great and quick feedback. Can you create an npm version with this merged? or do you plan to release a bigger release later?

**Comment by genedwards** [comment]
verison 0.0.10 published!

**Pull request merged**